test/eredis_tests.erl

```diff
@@ -203,6 +201,7 @@ tcp_closed_rig(C) ->
     %% closed. This behavior can be observed when Redis closes an idle
     %% connection just as a traffic burst starts.
     Socket = eredis_client:get_socket(C),
+    timer:sleep(250),
```

Comment:
^ This looks like a workaround to some nondeterministic condition... is there a better way to do it?

Reply:
Not unless I spend many minutes trying to figure out how to solve a bug that was already there.

Reply:
If you're satisfied that this can remain as-is (not caused by my pull request, which is a consequence on mostly poor past maintenance), you're welcome to Resolve conversation.

Reply:
Good enough for me.